tools/AzureRM.Example.psm1

```diff
@@ -38,6 +38,10 @@ if ($PSVersionTable.PSVersion.Major -ge 5)
         Register-ArgumentCompleter -CommandName $_.Command -ParameterName $_.Parameter -ScriptBlock $sb
     }
 }
+else
+{
+    %PSVersionDeprecationMessage%
```

Comment:
We need to save a file in their user directory to indicate that we have displayed this message, simila to the pattern we use for enabling/disabling data collection

Reply:
@markcowl is this what you had in mind?